_data/food.yml

```
@@ -34,3 +34,10 @@ websites:
       tfa: No
       twitter: tacobell
       facebook: tacobell
+ 
+    - name: Zomato
+      url: https://www.zomato.com
+      img: zomato.png
+      tfa: No
+      twitter: zomato
```

Just noticed that they spell their twitter handle with a capital Z.
```suggestion
      twitter: Zomato
```